opengrok-indexer/src/test/java/org/opengrok/indexer/history/GitRepositoryTest.java

```diff
@@ -207,6 +206,41 @@ public void testDetermineBranchAfterChange() throws Exception {
         }
     }
 
+    @Test
+    public void testGetHistoryInBranch() throws Exception {
```

Comment:
Just a note, with JUnit5 the methods no longer have to be public and using "package private"/default access is recommended (at least by Sonar :D ).

Reply:
Good to know, thanks !